docs/devices-api/more-info/webhooks.mdx

```diff
@@ -5,8 +5,7 @@ description: "Receive user health data as soon as it becomes available."
 
 ## Overview
 
-When using the Devices API, Metriport will send Webhook messages containing [user data](#user-data-message) to your app
-from [our supported Providers](/devices-api/more-info/our-integrations), as soon as the data becomes available.
+When using the Devices API, Metriport will send Webhook messages containing new [provider subscriptions](#user-provider-subscription) for each user, as well as [user data](#user-data-message) to your app from [our supported Providers](/devices-api/more-info/our-integrations), as soon as the data becomes available.
```

Comment:
nit: best to be consistent when writing technical documentation - is this "new provider subscriptions", or is it "new provider connection data", as referenced below? Let's pick one and stick to it.

My 2 cents would be for "new provider connections" - as the terminology we use elsewhere is that the user "connects" to providers, not "subscribes"